api/management/commands/sync_molnix.py

```diff
@@ -242,6 +266,14 @@ def sync_deployments(molnix_deployments, molnix_api, countries):
         personnel.country_to = country_to
         country_from = None
         personnel.surge_alert = surge_alert
+        personnel.appraisal_score = appraisal_score
```

Comment:
@szabozoltan69 many apologies - after discussing with Jonathan, it was decided that we should not store `appraisal_score` but just an `appraisal_received` boolean field. Sorry about this.

So here, we can just do something like `personnel.appraisal_received = len(md['appraisals']) > 0`